### PR TITLE
Prepare code base for Netlify deployments

### PR DESCRIPTION
**Pull request by lzrski**
*Re-opening #18 from local branch (i.e. not my fork) - otherwise CI and Netlify won't work*

Ressurect the `website/` directory. This is what is going to be deployed to Netlify.

To make it work we need to set some things in Netlify and GitHub and I don't have permissions for that. We can do it together as soon as you have time for a call.

Note: after the setup some things need to be updated in the code:

- [x] Set `siteId` in `.netlify/state.json` to correct values
- [x] Set the URLs in the ReadMe badge

**Personal note about Netlify**

I am really impressed by the service of Netlify, but I won't be using it for my own projects and I would advise you against it too. The terms of service seem really shady and can set you up for serious legal trouble. Consider the following scenario: they screw up and leak some sensitive personal information from your apps. Under the GDPR you may be subject to very high penalties from the EU (possibly millions of €). Normally you could claim it from a faulty contractor as damages, but check this out:

> 9. Limited Liability
>
> NETLIFY’S AGGREGATE LIABILITY FOR ALL CLAIMS, LOSSES, LIABILITIES OR DAMAGES IN CONNECTION WITH THIS AGREEMENT, YOUR ACCESS TO OR USE OF, OR YOUR INABILITY TO ACCESS OR USE, THE WEBSITE OR ANY SOFTWARE, SERVICES OR CONTENT ON THE WEBSITE, WHETHER AS A RESULT OF BREACH OF CONTRACT, TORT (INCLUDING NEGLIGENCE) OR OTHERWISE, REGARDLESS OF THE THEORY OF LIABILITY, IS LIMITED TO NO MORE THAN THE LESSER OF (A) THE TOTAL FEES PAID BY YOU IN THE THEN-CURRENT MONTH OR (B) ONE HUNDRED DOLLARS (US $100).
>
> IN ADDITION, NETLIFY WILL NOT UNDER ANY CIRCUMSTANCES BE LIABLE FOR LOST PROFITS, CONSEQUENTIAL, INDIRECT, PUNITIVE, EXEMPLARY OR SPECIAL DAMAGES.

*from https://www.netlify.com/legal/terms-of-use/*

That's just an example. There is more shit like this. Perhaps it is a very likely scenario, but this kind of clauses really discourage me from doing business with people who even suggest them in their contracts - on principle.

But I'm a very principled person and I won't judge you if you decide otherwise. From purely technical perspective Netlify seems excellent.

### CODE REVIEW

**Comment by lzrski** [comment]
I think it's ready. First deployment failed, but after restarting with cache cleaned it went fine. So IMO it's ready to merge, I just need your approval @zupo

**Comment by zupo** [comment]
Are there alternatives to Netlify? https://zeit.co/ maybe?

**Comment by zupo** [comment]
@lzrski: I asked a legal expert for help and he said that you take a look at Netlify's GDPR page (https://www.netlify.com/gdpr/), as well as their data processing agreement (https://www.netlify.com/legal/netlify-dpa.pdf). They are both quite detailed, and include additional data protection and data security obligations on their part, thereby altering the general terms of service to which you link. In their view, these additional assurances allow them to be used as a processor (hosting provider) under the GDPR. 

Then there is also https://law.stackexchange.com/questions/1410/do-warranty-disclaimers-in-software-licenses-carry-any-legal-weight and that AWS and others have basically similar or worse claims in their terms (https://aws.amazon.com/agreement/) making Netlify one of the better options regarding GDPR related issues.

**Comment by lzrski** [comment]
True. I was just looking at Zeit and GitLab and they have similar bullshit there. I find it really outrageous. IMO it's OK for a free software that we install and operate on our own machines - then it's our responsibility how we use it. But these are paid, proprietary **services** where operations are performed by their employees or contractors. As clients we really have no control or even knowledge about who can access or manipulate our and our customers' data. Pushing these provisions is a very unfair way of doing business. But you are right - seems like an "industry standard".

Anyway, IMO the code is good to go. I leave the decision about business (and moral?) judgement to you.

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+11
```
Comment (zupo) [inline .nvmrc]: What is this?

Comment (zupo) [reply .nvmrc]: Ah: https://github.com/creationix/nvm/pull/1047/files

**Comment by zupo** [comment]
Closing in favor of #21. Reason:

659f9ce0dc11af14aa10d0c927691de0

**Pull request closed without merging**